### ISSUE TITLE
Appium 1.6.4 can't run mobile web tests on Android Emulator's Native "Browser"

### ISSUE THREAD

**Issue by kodayashi** [appium/appium#8310]
## The problem

I can't run mobile web tests on Android Emulators with Appium 1.6.4. I've specified `Browser` as the `browserName`, but Appium starts chromedriver 2.28, then complains about Chrome not being installed. Can we not run tests against the native browser anymore?

## Environment

* Appium version (or git revision) that exhibits the issue: v1.6.4
* Last Appium version that did not exhibit the issue (if applicable): Unknown
* Desktop OS/version used to run Appium: macOS 10.12
* Node.js version (unless using Appium.app|exe): N/A
* Mobile platform/version under test: Android 6.0.0 Emulator (Genymotion)
* Real device or emulator/simulator: Emulator
* Appium CLI or Appium.app|exe: Appium Desktop v1.0.0

## Details

I tried pointing to my homebrew chromedriver 2.29 to see if that was the issue but got the same error. I see the Browser app opening, but then it's closed immediately.

## Link to Appium logs

https://gist.github.com/kodayashi/69bec53cc9158ccf025d83a1787ed9c7

## Code To Reproduce Issue [ Good To Have ]

1. Create an Android 6.0.0 Emulator in Genymotion (latest version).
2. Run a basic mobile web appium test against the native browser. The capabilities I used are in the gist.

**Comment by mykola-mokhnach** [comment]
Either update the system webview or downgrade chrome driver.
Closed as duplicate of https://github.com/appium/appium/issues/7138

**Comment by kodayashi** [comment]
Thanks @mykola-mokhnach, since I use chromedriver for desktop chrome tests, I have decided not to downgrade my local version.

Instead, I downloaded the google play APK from http://opengapps.org/ and flashed it on Nexus 5X and Nexus 9 emulators on Android 7.0 and 7.1 respectively.

Now I can just launch Google Chrome with the `"browserName" : "Chrome"` capability, and all works well with chromedriver 2.28.

I think this is a good path to take since all of the newer Android Emulators from Android Studio don't even include the "Browser" app anymore. Also, that webview app may behave differently than chrome itself (though I'm not sure).

Thanks for the link to the original issue!
